### DISCUSSION TITLE
Links with descriptions in forge-topic-mode

### DISCUSSION THREAD

**Discussion by agzam** [magit/forge#753] [Ideas]
Would be nice if in addition to the bare urls, links also included their descriptive titles. It's hard to tell what's that about when you see something like: "related: https://github.com/foo/bar/issues/5244"

**Comment by tarsius** [comment]
It's a little hard to catch up to github, after all they are more than one person. :]

**Comment by tarsius** [comment]
I no longer plan to implement this eventually.
